Source/cg_SectionGeneralSettings.cpp

```diff
@@ -83,7 +211,8 @@ SectionGeneralSettings::SectionGeneralSettings(GrisLookAndFeel & grisLookAndFeel
 
     mNumOfSourcesEditor.setExplicitFocusOrder(2);
     mNumOfSourcesEditor.setText("2");
-    mNumOfSourcesEditor.setInputRestrictions(1, "12345678");
+    auto const MAX_NUMBER_OF_SOURCES { juce::JUCEApplicationBase::isStandaloneApp() ? 128 : 8};
```

Comment:
I think it could be raised to 256, I have seen David create 256-size channel aggegates